roles/general/tasks/swapfile.yml

```diff
@@ -16,6 +16,12 @@
 - name: Put the swap entry into fstab
   mount: name=none src={{ swap_file.path }} fstype=swap opts=sw passno=0 dump=0 state=present
 
+- name: Check swapon swapfile
+  raw: swapon | grep "{{ swap_file.path }}"
```

Comment:
I might should use `shell` -- `command` was failing b/c of the pipe character. I'll see if `shell`will do.